examples/drivers/windows/otApi/otApi.cpp

```diff
@@ -1522,14 +1522,15 @@ otThreadGetExtendedPanId(
 }
 
 OTAPI 
-void 
+ThreadError 
 OTCALL
 otThreadSetExtendedPanId(
     _In_ otInstance *aInstance, 
     const uint8_t *aExtendedPanId
     )
 {
-    if (aInstance) (void)SetIOCTL(aInstance, IOCTL_OTLWF_OT_EXTENDED_PANID, (const otExtendedPanId*)aExtendedPanId);
+    if (aInstance == nullptr) return kThreadError_InvalidArgs;
+    return DwordToThreadError(SetIOCTL(aInstance, IOCTL_OTLWF_OT_EXTENDED_PANID, (const otExtendedPanId*)aExtendedPanId));
```

Comment:
You need to also update [iocontrol.c](https://github.com/jwhui/openthread/blob/0455c96df601a4cf75c88ca82ff53dbd2a73bdbf/examples/drivers/windows/otLwf/iocontrol.c#L1247) to pass the error from OpenThread back; similar to [this](https://github.com/jwhui/openthread/blob/0455c96df601a4cf75c88ca82ff53dbd2a73bdbf/examples/drivers/windows/otLwf/iocontrol.c#L1147).